openwisp_utils/commitizen/openwisp.py

```
@@ -0,0 +1,181 @@
+            "[feature] Add commit convention enforcement #110\n\n"
+            "Introduce a Commitizen-based commit workflow to standardize\n"
+            "commit messages across the OpenWISP project.\n\n"
+            "Fixes #110"
+        )
+
+    def schema(self) -> str:
+        return "[<type>] <Title>"
+
+    def schema_pattern(self) -> str:
+        # Allow merge commits (starting with "Merge") or regular commits with prefix
+        return r"(?sm)^(?:Merge .*|\[[a-z0-9!/:-]+\] [A-Z][^\n]*( #(?P<issue>\d+))?$(\n\n.+\n\nFixes #(?P=issue)\n?)?)$"  # noqa: E501
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)